demo/db.json

```diff
@@ -47,6 +47,30 @@
 		"invalid": true,
 		"functional_group": "Vacuum",
 		"location_group": "S 2"
+	},
+	"TV5L0-VGC-1": {
```

Comment:
What's up with this extra device in general? What is it used for?

Reply:
This is what I meant. I don't know where this device came from, but I guess its part of the demo database so I didn't  look too much into it.